packages/DataHeader/stories/DataHeaderWithBackground.datagrid.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
```

Comment:
This seems too similar to the `DataHeader.datagrid.stories.js` story.  We can probably eliminate one of them.

Reply:
I still see both... did you mean to leave them both still?